truffle-config.js

```diff
@@ -1,46 +1,53 @@
-const truffleConfig = require('@gnosis.pm/util-contracts/src/util/truffleConfig')
+const truffleConfig = require("@gnosis.pm/util-contracts/src/util/truffleConfig");
 
-const DEFAULT_GAS_PRICE_GWEI = 25
-const DEFAULT_GAS_LIMIT = 6721975
-const DEFAULT_MNEMONIC = 'candy maple cake sugar pudding cream honey rich smooth crumble sweet treat'
+const DEFAULT_GAS_PRICE_GWEI = 25;
+const DEFAULT_GAS_LIMIT = 6721975;
+const DEFAULT_MNEMONIC =
+  "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
 
 // Load env vars
-require('dotenv').config()
+require("dotenv").config();
 
 // Get the mnemonic
-const privateKey = process.env.PK
-let mnemonic = process.env.MNEMONIC
+const privateKey = process.env.PK;
+let mnemonic = process.env.MNEMONIC;
 if (!privateKey && !mnemonic) {
-  mnemonic = DEFAULT_MNEMONIC
+  mnemonic = DEFAULT_MNEMONIC;
 }
 
 // Solc
-let solcUseDocker = process.env.SOLC_USE_DOCKER === 'true' || false
-let solcVersion = '<0.5.5'
+let solcUseDocker = process.env.SOLC_USE_DOCKER === "true" || false;
+let solcVersion = "<0.5.7";
```

Comment:
Should we take this out completely?

Reply:
I think it is needed for the logic of the truffleMigrate scripts that we are importing. I think that should be all refactored later.

Reply:
Otherwise, if I do not hand over a parameter, it will default to truffle 4 versions